backend/src/models/reports/Tab_34_rpt_PA_StatusDashboard.js

```diff
@@ -1,17 +1,58 @@
 const dbConnection = require("@database/databaseConnection");
 const { knex } = dbConnection();
+const _ = require("lodash");
 
 /**
  * Gets data for the Divisional Project Reports - Project Dashboard report.
  *
+ * @param   {number[]} portfolios Optional list of portfolio_ids to limit report to. If empty, returns data for all portfolios.
  * @returns {any[]}
  */
-const Tab_34_rpt_PA_StatusDashboard = () => {
-  const query = knex().raw();
+const Tab_34_rpt_PA_StatusDashboard = (portfolios) => {
+  const portfolioArray = _.castArray(portfolios);
+  const isValidPortfolioList = portfolioArray.every((p) => p);
```

Comment:
could this just be an array of true's and still pass... just making sure that we do enough validation to ensure this doesn't get set when it shouldn't.   I't is probably ok, but just double checking.  Maybe look where this gets called and ask yourself is there anyway the wrong data could still get by here.

Reply:
It could be an array of trues, and that would pass the validator, though it would 

If you like, I can do something like:
```
portfolioArray.every(_.isNumber)
```